### PR TITLE
Objects/scene-objects/group

### PR DESCRIPTION
**Pull request by kwakuTM**
Updated Objects/scene-objects/group documentation

Note: Updated Images are not reflected in the PR, but if you switch to my branch you can see them.

### CODE REVIEW

File: objects/scene-objects/group.md [hunk 1]
```diff
@@ -1,21 +1,7 @@
 # Group
 
-The primary function of the **Group** **Node** is that it allows us to transform multiple **Objects**, relative to a single **Pivot Point**, as if though they are a single, unified **Object**.
+The primary function of the **Group** **Object** is that it allows us to transform multiple **Objects**, relative to a single **Pivot Point**, as if though they are a single, unified **Object**.
```
Comment (polygami) [inline objects/scene-objects/group.md:3]:
```suggestion
The **Group** **Object**  works like a folder, as it allows you to structure and organize **Objects**, making **Scenes** with high amounts of different **Objects**, more manageable.

Another useful function of the **Group** **Object** is that it allows us to transform multiple **Objects**, relative to a single **Pivot Point**, as if they were are a single **Object**.
```

File: objects/scene-objects/group.md [hunk 2]
```diff
@@ -1,21 +1,7 @@
...
 
 Additionally, it is a useful way of organizing collections of **Objects** into categories, and making **Scenes** with high amounts of different **Objects**, more manageable.
```
Comment (polygami) [inline objects/scene-objects/group.md:5]: Remove this as I merged it into the above suggestion

File: objects/scene-objects/group.md [hunk 3]
```diff
@@ -1,21 +1,7 @@
...
-
-# Usage
-
-![The Get Material Node.](../../../incari-doc/.gitbook/assets/objects/scene-objects/group/group.gif)
-
-# See Also
-- [**Camera**](camera.md)
-- [**List**](list.md)
\ No newline at end of file
+Like any other **Scene Object**, we can manipulate the **Position**, **Rotation** and **Scale** of a **Sprite** using the **Transformation Attributes** and adjusting its **Rotation Pivot**.
```
Comment (polygami) [inline objects/scene-objects/group.md:7]:
```suggestion
Like any other **Scene Object**, we can manipulate its **Transformation Attributes** and adjust its **Rotation Pivot**.
```

**Review by polygami — CHANGES_REQUESTED** [review]

File: objects/scene-objects/group.md (hunk 2 above)
Comment (kwakuTM) [reply objects/scene-objects/group.md:5]: you mean
*Additionally, it is a useful way .... ... *

**Pull request merged**